ConwaysGameOfLife/GameController.cs

```diff
@@ -103,11 +138,11 @@ public string CreateValidLocationString()
             return input;
         }
 
-        public Cell SetLiveCellLocation(Location location)
+        public Universe ReturnUniverseAfterSettingLiveCellLocation(Location location)

```

Comment:
this method is doing 2 things - changing the cell state and returning a universe. recommend to separate these concerns & rename the method